src/MXFTypes.h

```diff
@@ -688,29 +689,19 @@ namespace ASDCP
 	  const char* EncodeString(char* str_buf, ui32_t buf_len) const;
 	};
 
+      //
       class J2KExtendedCapabilities : public Kumu::IArchive
         {
         public:
-	  i8_t Pcap;
-	  i8_t Ccapi[16]; // this is certainly wrong, need the spec
+	  ui8_t Pcap; // TODO: is this the right type?
```

Comment:
@jhursty ui32_t

Reply:
@palemieux palemieux Big-endian, yes?

Reply:
`(Pcap >> 31) & 0x1` is `Pcap(1)`

See Table A.11ter at https://www.itu.int/rec/T-REC-T.800-201906-I/en

Reply:
> Big-endian, yes?

It should be written to MXF like any 32-bit integer, so, yes, big endian.